packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -310,13 +310,25 @@ class AuroraClusterEngine extends MySqlClusterEngineBase {
  * https://docs.aws.amazon.com/AmazonRDS/latest/AuroraMySQLReleaseNotes/Welcome.html
  */
 export class AuroraMysqlEngineVersion {
-  /** Version "5.7.12". */
+  /**
+   * Version "5.7.12".
+   * @deprecated  Legacy Version no longer supported by Amazon RDS. Migrate to the available version.
```

Comment:
```suggestion
   * @deprecated  Version 5.7.12 is no longer supported by Amazon RDS. 
```

Adding this suggestion in the one place but the pattern should be followed for each of these. `Migrate to the available version.` isn't a helpful message here so we might as well omit it if we don't want to specify what it is, given the potential upkeep of that info.

Reply:
Thank you very much. I have generally reviewed all the messages with the policy you mentioned.